lib/engine/part/path.rb

```diff
@@ -5,12 +5,16 @@
 module Engine
   module Part
     class Path < Base
-      attr_reader :a, :b, :branch, :city, :edges, :junction, :node, :offboard, :stop, :town, :terminal
+      attr_reader :a, :b, :branch, :city, :edges, :junction, :node, :offboard,
```

Comment:
we should not do this, if paths have multiple X, there's no reason to store just one

you should replace all old things with plurals, stop and stops is wrong

so let's make everything

branches, cities, junctions nodes etc

Reply:
Sorry, I was trying to be conservative and avoid touching more files. I was going to clean this up when I visited routing. I'll fix it up now.

Reply:
i'd prefer more aggressive clean up